Ginger/Ginger/Run/RunSetsExecutionsHistoryPage.xaml.cs

```diff
@@ -763,6 +841,31 @@ private void ExportUseCaseFromBusinessFlowExecutionSequence(BusinessFlowExecutio
             ExportUseCaseFromBusinessFlow(executedBF);
         }
 
+        private string GenerateVirtualGroupNameForBusinessFlow(BusinessFlow bf)
```

Comment:
Please check if the group name exist in activity, if not, then only use the virtual group name

Reply:
group name exists in both remote and lite db exeuction data and the new code is now referring the group name from there.